frontend/src/Components/EditableFields.tsx

```diff
@@ -22,10 +22,8 @@ export const EditableField = ({
   workstation,
   fieldKey,
   placeholder,
-  GroupSelect,
   isEven,
 }: EditableFieldProps & {
-  GroupSelect: GS;
```

Comment:
Yeah removing this is probably fair. I added it to support group selection being a drop-down rather than pure text input, and I know at one point I had the intention to go back to drop-downs but I think it's probably not worth doing now.